ganga/GangaCore/Runtime/GPIFunctions.py

```diff
@@ -93,3 +96,22 @@ def convert_merger_to_postprocessor(j):
         mp.process_objects.append(stripProxy(j).merger)
         stripProxy(j).postprocessors = mp
 
+def runfile(path_to_file):
+    """
+    A wrapper for the runpy.run_path() function.
+    Usage:
+    Ganga In[]: runfile('myfile.py')
+    Note: If you want to run a file which includes something like:
+
+    j = Job()
+    j.submit()
+
+    then you have to run the function with init_globals=globals() in the Ganga interpreter like this
+
+     Ganga In[]: runfile('myfile.py', init_globals=globals());
```

Comment:
The function no longer takes the `init_globals` argument, so this line should be fixed.